src/pages/interim/_cta-binary.js

```diff
@@ -16,7 +24,7 @@ const CtaBinary = ({ is_white }) => {
                 <Localize
                     translate_text="No sign up needed. Log in with your <0>Binary.com</0> credentials."
                     components={[
-                        <LocalizedLinkText
+                        <BinaryLink
```

Comment:
CtaBinary is used several times throughout the interim page, please change the hover to white when the props is white is true